app/controllers/admin_user_controller.rb

```diff
@@ -22,18 +22,26 @@ class AdminUserController < AdminController
   def index
     @query = params[:query]
 
+    @roles = params[:roles] || []
     @sort_options = index_sort_options
 
     @sort_order =
       @sort_options.key?(params[:sort_order]) ? params[:sort_order] : 'name_asc'
 
     users = if @query
```

Comment:
Now that we are using the search button to apply a filter, this would work better as `users = if @query.present?` otherwise it will always add the extra WHERE clauses, even when the submitted query is blank